tests/test_outliers/test_winsorizer.py

```diff
@@ -164,6 +164,26 @@ def test_quantile_capping_both_tails_with_fold_15_percent(df_normal_dist):
     assert np.round(df_normal_dist["var"].max(), 3) > np.round(0.11823196128033647, 3)
 
 
+def test_indicators_are_added(df_normal_dist):
```

Comment:
could we add an additional test to test this class when the user selects only 1 or 2 variables from the dataframe to be capped?

This will flag issue about adding indicators only to those variables and not to the entire dataset.